### PR TITLE
HomePage Nav Donate Button

### PR DESCRIPTION
**Pull request by VickyAbishek**
Added Donate button to the homepage nagvigation bar pointing to the donate url

Feature

# What kind of change does this PR introduce
Added donate button

## What is the current behavior
No Donate button

## What is the new behavior
Added Donate button

### Checklist for this PR

* [x] I have read [contributing guidelines](https://github.com/coderplex/coderplex/blob/develop/.github/CONTRIBUTING.md). 
* [ ] I executed `yarn lint` locally to format my code.
* [ ] I added revelant Tests to verify my code.
* [] All tests `yarn test` ran successfully.
* [x] Pull request is towards `develop` branch.
* [ ] I have tested responsiveness of the UI by checking on devices using [ScreenFly](http://quirktools.com/screenfly/) 
* [ ] Added myself to contributors table 
* [ ] Documentation
* [x] Ready to be merged

### CODE REVIEW

File: components/common/header/index.styles.js [hunk 1]
```diff
@@ -107,6 +107,11 @@ export const NavLink = styled.li`
         }
       }
     }
+    &.donate_btn {
+      background: #e91e63;
+      color: white;
+      border-radius: 10px;
```
Comment (vinaypuppal) [inline components/common/header/index.styles.js]: Apply these changes:
```suggestion
	border-radius: 6px;
	font-weight: bold;
	line-height: inherit;
```

**Comment by KirankumarAmbati** [comment]
One more suggestion: 

I think it is better to have Donate button color same as our theme color instead of red.

**Comment by vinaypuppal** [comment]
One more thing I forgot above, make the button look good on mobile. For example on mobile it should like this:
coderplex-xiplocrtgx now sh_(Nexus 5X)

**Comment by duttakapil** [comment]
I think the button height here is slightly too tall. It should be more wider on the sides / narrow horizontally

**Comment by vinaypuppal** [comment]
@duttakapil Yeah, my suggested CSS changes above make it look good.

**Comment by VickyAbishek** [comment]
Hi all,
Thank you all the for the suggestions. Created another pull request with the changes.
This is my first PR. Just bare me for the first few commits 😃, I will make sure I won't repeat the mistakes in the future commit.

**Comment by VickyAbishek** [comment]
> @VickyAbishek Looks good!
> Few suggestions:
> 
> 1. Please check the suggested changes above!
> 2. Add a hover effect to the button as we have for other buttons on the site.
> 3. Clicking on button should open donation page in a new tab!

Done.

**Comment by duttakapil** [comment]
@VickyAbishek you don't need to create a new PR. You just push your code to the branch in your fork from which you sent the first PR, and it automatically updates

**Comment by vinaypuppal** [comment]
@all-contributors please add @VickyAbishek for code

**Pull request merged**